test.sh

```diff
@@ -50,7 +50,7 @@ function setup_osbs() {
 
   # List common install dependencies
   PKG_COMMON_EXTRA=(git-core gcc krb5-devel "$PY_PKG-devel" popt-devel redhat-rpm-config
-                    cairo-devel gobject-introspection-devel cairo-gobject-devel)
+                    cairo-devel gobject-introspection-devel cairo-gobject-devel ostree)
```

Comment:
actually, we did have it before and it was removed by https://github.com/containerbuildsystem/atomic-reactor/commit/75a4fa9cb766c49663f392c15ab24b0e5486f73d. My guess is it didn't fail till now since we were skipping flatpak tests.

Reply:
Ah, fair enough, let's keep it then :+1: